code-challenges.test.js

```diff
@@ -15,42 +15,173 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// --- > ::STUDENT NOTES:: This question confused me at first because this problem was not supplied with variables like it states in the test question/statement, only suggested example inputs? With that said, I bypassed this statement and proceeded to make the test work as I suspect I am supposed to.
+
+describe("sequencer", () => {
+    it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+        expect(sequencer(6)).toEqual([1, 1, 2, 3, 5, 8])
+        expect(sequencer(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
+
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
+        // JEST "GOOD" TEST FAIL
+        // -------------------------
+        // FAIL./ code - challenges.test.js
+        // sequencer
+        //     ✕ takes in a number(greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence(7 ms)
+
+        //   ● sequencer › takes in a number(greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence
+
+        // ReferenceError: sequencer is not defined
+
+
+
 // b) Create the function that makes the test pass.
 
+// PSEUDO CODE
+// ----------------------
+// Create a FUNCTION called SEQUENCER that takes in a NUMBER > 2
+// Create a VARIABLE to hold an ARRAY of first 2 values of fibonacci sequence
+// Per established rules of the fibonacci sequesnce, ITERATE the number of times in the paramaters argument
+// Fill the newly created variable with the fibonacci sequence via the PUSH METHOD.
+
+
+const sequencer = (number) => {
+    let array = [1, 1]
+    for (let i = 0; i < number - 2; i++) {
+        if (number > 2) {
+            array.push(array[i] + array[i + 1])
+        }
+    }
+    return array
+}
```

Comment:
This looks great!  Since you took into account an edge case in this code, you are always welcome and even encouraged to test those edge cases with your test as well!